control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1376,6 +1376,15 @@ func (r *HostedControlPlaneReconciler) reconcileManagedEtcd(ctx context.Context,
 		r.Log.Info("reconciled etcd servicemonitor", "result", result)
 	}
 
+	pdb := manifests.EtcdPodDisruptionBudget(hcp.Namespace)
```

Comment:
We also need to watch PDBs now and add the annotation so chances in the PDB trigger us or not?

Reply:
Updated to add the watch on PDB and also updated reconcile to apply the owner reference to the PDB during reconcile. Thanks!